echopype/tests/conftest.py

```diff
@@ -28,6 +28,16 @@ def test_path():
     }
 
 
+@pytest.fixture(scope="session")
```

Comment:
From my understanding, you moved `ek60_test_data` here because it was used within `test_echodata_combine` and `test_zarr_combine`. However, this fixture seems out of place. Also, in the future we may want to have  `ek60_test_data` be different for `test_echodata_combine` and `test_zarr_combine`. Would it be better to just copy this into both `test_echodata_combine` and `test_zarr_combine`?